projects/observability/src/shared/components/explore-query-editor/explore-query-editor.component.test.ts

```diff
@@ -87,6 +89,10 @@ describe('Explore query editor', () => {
         getAutoDurationFromTimeDurations: () => new TimeDuration(15, TimeUnit.Second),
         getAutoDuration: () => new TimeDuration(15, TimeUnit.Second)
       }),
+      mockProvider(FeatureStateResolver, {
+        getCombinedFeatureState: () => of(FeatureState.Disabled),
+        getFeatureState: () => of(FeatureState.Disabled)
```

Comment:
we probably only need to override one of the two methods.

Reply:
removed combinedFeatureState